scl/windowseventlog/plugin.conf

```diff
@@ -0,0 +1,20 @@
+
+block parser windows-eventlog-parser() {
+    channel {
+        parser {
+            db-parser(file("windowseventlog.xml") drop-unmatched(yes));
+        };
+    };
+};
+
+template t_wineventlog_json {
+    template("$(format-json --scope nv-pairs --exclude MESSAGE \
+                --exclude SOURCE \
+                win.EventRecordID=${.SDATA.win@18372\.4.EVENT_REC_NUM} \
+                win.TimeCreated=${ISODATE} \
+                win.Channel=${.SDATA.win@18372\.4.EVENT_NAME} \
+                win.EventID=${.SDATA.win@18372\.4.EVENT_ID} \
+                PATTERN_CLASS=${.classifier.class} \
+               )\n");
```

Comment:
> You still have the template here, what's the use-case, why is that needed here?

by @bazsi, from https://github.com/balabit/syslog-ng/pull/1572#discussion_r138559874

Reply:
I've talked to Gergo and agreed that the template would be removed, with that this is ready to be merged.

Reply:
As requested removed the template from the config